tests/conftest.py

```diff
@@ -248,7 +251,7 @@ def response(*args, **kwargs):
 
                 if function_name == "auth_mapping":
                     if not known_user:
-                        raise ArboristError("User does not exist in Arborist")
+                        raise ArboristError("User does not exist in Arborist", 401)
```

Comment:
I had to add this to fix an issue with `test_arborist_unknown_user`, which was failing because of this missing `code` parameter in the `ArbotristError` constructor both in Travis and on my localhost.

```
  File "/peregrine/tests/conftest.py", line 254, in response
    raise ArboristError("User does not exist in Arborist")
```

None of the test code or application code related to this had changed in the past two years or so. I'm 99% sure this is because, by using Poetry, we're moving up to Python 3.7 and it doesn't like this explicit parameter not being there. We're using 3.7 because that's what I see in almost all our other `pyproject.toml` files and I'd rather be consistent with that.

Reply:
probably because of [this fix](https://github.com/uc-cdis/gen3authz/pull/19) 😄